src/Pack.js

```diff
@@ -0,0 +1,194 @@
+/**
+    @external Viz
+    @see https://github.com/d3plus/d3plus-viz#Viz
+*/
+import {nest} from "d3-collection";
+import {hierarchy, pack} from "d3-hierarchy";
+
+import {accessor, assign, configPrep, constant, elem} from "d3plus-common";
+import * as shapes from "d3plus-shape";
+import {Viz} from "d3plus-viz";
+
+/**
+    @class Pack
+    @extends Viz
+    @desc Uses the [d3 pack layout](https://github.com/d3/d3-hierarchy#pack) to creates Circle Packing chart based on an array of data.
+*/
+export default class Pack extends Viz {
+
+  /**
+      @memberof Pack
+      @desc Invoked when creating a new class instance, and sets any default parameters.
+      @private
+  */
+  constructor() {
+
+    super();
+
+    this._layoutPadding = 1;
+    this._on.mouseenter = () => {};
+    this._on["mouseleave.shape"] = () => {
+      this._tooltip = false;
+      this.hover(false);
+    };
+
+    let hoverData = [];
+    const recursionCircles = d => {
+      if (d.values) {
+        d.values.forEach(h => {
+          hoverData.push(h);
+          recursionCircles(h);
+        });
+      }
+      else {
+        hoverData.push(d);
+      }
+    };
+    const defaultMouseMove = this._on["mousemove.shape"];
+    this._on["mousemove.legend"] = (d, i) => {
+      const ids = this._ids(d, i);
+      hoverData = [d];
+      recursionCircles(d);
+
+      this.hover(h => {
+        const _hover = Object.keys(h).filter(key => key !== "value").every(key => d[key] && d[key].includes(h[key]));
+
+        if (_hover) hoverData.push(h);
+        else if (ids.includes(h.key)) {
+          hoverData.push(h);
+          recursionCircles(h);
+        }
+
+        return hoverData.includes(h);
+      });
+
+    };
+    this._on["mousemove.shape"] = (d, i) => {
+      defaultMouseMove(d, i);
+
+      this._tooltip = d.__d3plusTooltip__;
```

Comment:
Never overwrite a user-definable variable like this. The use is able to set `this._tooltip = false` by using `.tooltip(false)`, and this would interfere with that.

Maybe try this?
```js
if (d.__d3plusTooltip__) defaultMouseMove(d, i);
```